lib/methods/errors/ResponseError.js

```diff
@@ -1,25 +1,43 @@
 const DAPIClientError = require('../../errors/DAPIClientError');
 
-/**
- * @abstract
- */
 class ResponseError extends DAPIClientError {
   /**
+   *
    * @param {number} code
    * @param {string} message
+   * @param {object} metadata
```

Comment:
Object

Reply:
This was fixed by ESlint
We have such setting in the project